Tests/Core/CancellableErrorTests.swift

```diff
@@ -1,5 +1,6 @@
 import Foundation
 import PromiseKit
+import TestsShared
```

Comment:
What is this new module for?

Reply:
The new module is to have a common set of files for tests targets, i.e. `TestError` is there so you do not need to declare dummy errors everywhere as it makes things confusing and tests harder to read.